docs/GettingStarted/configuration/ExporterCommittime.md

```diff
@@ -13,70 +22,223 @@ Currently we support GitHub, GitLab, BitBucket, Gitea and Azure DevOps Git servi
-| `LOG_LEVEL`                  | no       | `git`, `image`     | Set the log level. One of `DEBUG`, `INFO`, `WARNING`, `ERROR`                                                                                                            | `INFO`                                                                    |
-| `APP_LABEL`                  | no       | `git`, `image`     | Changes the label key used to identify applications                                                                                                                      | `app.kubernetes.io/name`                                                  |
-| `NAMESPACES`                 | no       | `git`              | Restricts the set of namespaces from which metrics will be collected. ex: `myapp-ns-dev,otherapp-ci`                                                                     | unset; scans all namespaces                                               |
-| `PELORUS_DEFAULT_KEYWORD`    | no       | `git`, `image`     | ConfigMap default keyword. If specified it's used in other data values to indicate "Default Value" should be used                                                        | `default`                                                                 |
-| `COMMIT_HASH_ANNOTATION`     | no       | `git`, `image`     | Annotation name associated with the Build from which hash is used to calculate commit time                                                                               | `io.openshift.build.commit.id`                                            |
-| `COMMIT_REPO_URL_ANNOTATION` | no       | `git`, `image`     | Annotation name associated with the Build from which GIT repository URL is used to calculate commit time                                                                 | `io.openshift.build.source-location`                                      |
-| `COMMIT_DATE_ANNOTATION`     | no       | `image`            | Annotation name associated with the Image from which commit time is taken.                                                                                               | `io.openshift.build.commit.date`                                          |
-| `COMMIT_DATE_FORMAT`         | no       | `image`            | Format in `1989 C standard` to convert time and date found in the Docker Label `io.openshift.build.commit.date` or annotation for the Image                              | `%a %b %d %H:%M:%S %Y %z`                                                 |
+
+## Commit Time Exporter configuration options
+
+This is the list of options that can be applied to `env_from_secrets`, `env_from_configmaps` and `extraEnv` section of a Commit time exporter.
+
+Table below represents configuration options, which are valid for any Commit Time Exporter instance.
```

Comment:
I would remove this and just have one table, I believe the info in Required is enough